<launch>
  <!-- Launch server separatly (better) -->
  <!-- <include file="$(find vive_ros)/launch/server_vr.launch"> -->
    <!-- <arg name="moveit_warehouse_database_path" value="$(arg db_path)"/> -->
  <!-- </include> -->

  <node name="vive_pose" pkg="vive_ros" type="vive_pose" output="screen"/>

  <node name="panda1_control" pkg="moveit_vive" type="panda_dual_control">
    <param name="arm_id" type="str" value="panda1"/>
    <param name="controller_id" type="str" value="right"/>
  </node>

  <node name="panda2_control" pkg="moveit_vive" type="panda_dual_control">
    <param name="arm_id" type="str" value="panda2"/>
    <param name="controller_id" type="str" value="left"/>
  </node>

</launch>
 